kdc/kerberos5.c

```diff
@@ -1705,6 +1705,31 @@ add_enc_pa_rep(kdc_request_t r)
 			   KRB5_PADATA_FX_FAST, NULL, 0);
 }
 
+static krb5_error_code
+get_local_tgt(krb5_context context,
```

Comment:
Bike-shedding, but prefer get_local_krbtgt() or get_local_tgs() as you're not getting a ticket.